### ISSUE TITLE
Suggest Feature : Zoom In / Out Layout

### ISSUE THREAD

**Issue by bbagwang** [Viladoman/StructLayout#26]
this is a really good extension :)
Hope we can zoom in / out the layout.

Thanks!

**Comment by Viladoman** [comment]
Thanks! I am glad you like it. 
I want to add zoom at some point. It is already in the [UX projects section](https://github.com/Viladoman/StructLayout/projects/1#card-48491945). 

Hopefully I will have some spare time soon to do some work on this.

**Comment by BobbyAnguelov** [comment]
+1 for this please...

On a 4k Monitor with 125% scaling the UI is a little "large". I should definitely be able to see more than 12 rows.

![image](https://github.com/Viladoman/StructLayout/assets/10767490/bd1e9420-464c-4334-8ac0-3ef592431b59)

**Comment by Viladoman** [comment]
Hopefully I can get to it sooner than later. 

Meanwhile, for big structures like this I would suggest using the 'Flat' mode instead of 'Stack' ( the option is at the top of the Struct Layout window ). 

The stack mode adds this small padding for each nested node so it can be clicked back to collapse and shows the current nested state better. When there are a several levels of nested structures then these paddings can add up quite a lot. The 'Flat' mode is basically 0 padding and in order to collapse or go up 1 level the use of context menu on the mouse right click is mandatory. 

![Flat screenshot](https://github.com/Viladoman/StructLayout/wiki/data/Flat.png?raw=true)

versus

![Stack screenshot](https://github.com/Viladoman/StructLayout/wiki/data/Stack.png?raw=true)

TLDR: I agree that a zoom slider is long overdue.
